presto-main/src/main/java/com/facebook/presto/transaction/InMemoryTransactionManager.java

```diff
@@ -420,7 +421,8 @@ private synchronized CatalogMetadata getTransactionCatalogMetadata(ConnectorId c
                 catalogMetadata = new CatalogMetadata(
                         metadata.getConnectorId(), metadata.getConnectorMetadata(), metadata.getTransactionHandle(),
                         informationSchema.getConnectorId(), informationSchema.getConnectorMetadata(), informationSchema.getTransactionHandle(),
-                        systemTables.getConnectorId(), systemTables.getConnectorMetadata(), systemTables.getTransactionHandle());
+                        systemTables.getConnectorId(), systemTables.getConnectorMetadata(), systemTables.getTransactionHandle(),
```

Comment:
What do you think of @electrum comment recommending putting `metadata`, `informationSchema` and `systemTables` in separate lines?
https://github.com/prestodb/presto/pull/11561#discussion_r229530708

Reply:
Oh, this has been discussed. Leave as is.

This is the first time i see @electrum not following the general rules 😉

Reply:
I think @dain wrote this code originally. I am fine with putting each on a separate line :)

Reply:
Alright, to recap, how about we settle on one single universally applied standard.  I will be each on a separate line because it's too long on one line, and it is arbitrary to group them in this way.

Reply:
And let the record show the [Presto Development Guidelines](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting) specifically mentions this :)

```
Parameter lists are often too long to be readable on a single line. When wrapping parameters, put each one on a separate line. Otherwise, keep them all on the same line. This applies to both declarations and call sites.
```

Reply:
Thanks for the pointer. I didn't know we have such a document :)